### PR TITLE
feat: added cart-page & functionality

### PR DESCRIPTION
**Pull request by sharmayatendra**
# Changes Added

## Functionality
- add to the cart
- increment/decrement cart items
- remove from the cart

image

### CODE REVIEW

File: src/components/ProductCard/ProductCard.jsx [hunk 1]
```diff
@@ -19,15 +21,15 @@ return (
...
                 </div>
             </div>
         </div>
         <footer className="card-footer">
-            <div className="card-footer-btn">
-                <button className="card-btn">Add to Cart</button>
-                <button className="card-btn">Buy Now</button>
+            <div className="card-footer-btn add-cart">
+                {cartItems.find(ele => ele._id === product._id) ?<Link to="/cart"><button className="card-btn">Go to Cart</button></Link> : <button className="card-btn" onClick={addToCart} >Add to Cart</button>}
+                {/* <button className="card-btn">Buy Now</button> */}
```
Comment (tushargoyal812) [inline src/components/ProductCard/ProductCard.jsx:32]: remove commented code if its not needed

**Review by tushargoyal812 — COMMENTED** [review]
liked how you structure your code.
variable naming is good.
your code is to the point and clean

**Review by vikrantkode — APPROVED** [review]
All looks good to me. Ready to merge

File: src/components/ProductCard/ProductCard.jsx (hunk 1 above)
Comment (sharmayatendra) [reply src/components/ProductCard/ProductCard.jsx:32]: i left that intentionally will use that when implement checkout functionality

**Pull request merged**